.github/workflows/os.yml

```diff
@@ -91,9 +88,7 @@ jobs:
         with:
           ABI: ${{ matrix.ABI }}
       - name: Install digraphs-lib . . .
-        run: |
-          curl --retry 5 -L -O "https://digraphs.github.io/Digraphs/${{ env.DIGRAPHS_LIB }}.tar.gz"
-          tar xf "${{ env.DIGRAPHS_LIB }}.tar.gz"
+        run: git clone https://github.com/digraphs/digraphs-lib.git
```

Comment:
I guess we could also use `actions/checkout@v5` here, with the path `digraphs-lib`. But this seems fine.